### ISSUE TITLE
Return errors gracefully via API

### ISSUE THREAD

**Issue by malikkotob** [usdoj/foia.gov#355]
As a portal owner, in order to prevent the webform submission API from failing on large submissions, I need the memory limit handling to be improved.

> I seem to be getting an OOM error when using a 19mb file upload. The strange thing is, the response code is 200, but it _does_ kick off the xhr's onerror handler, so we treat it as a generic error. The response body is:

```
<br />
<b>Fatal error</b>:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 26913658 bytes) in <b>/mnt/www/html/foiadev/vendor/symfony/serializer/Encoder/JsonDecode.php</b> on line <b>91</b><br />
```

## Steps to reproduce the behavior

1. Go to http://foiafrontdev.prod.acquia-sites.com/request/agency-component/8216158f-8089-431d-b866-dc334e8d4758/
1. Fill out required form fields and include a file attachment that is at least 19MB
1. Submit the form

**Comment by vz3** [comment]
Would this be triggered if 2 people submit 10mb files or three submit 7mb files?

**Comment by adborden** [comment]
The lower limit seems to be 18MB. At 18MB, it fails consistently 3/3 submissions. At 17MB, I successfully submitted 3/3 requests.

**Comment by malikkotob** [comment]
Not quite, @vz3. Every request to the server is allocated it's own set of memory.

**Comment by malikkotob** [comment]
@vz3, @adborden, we increased the memory limit in the dev environment. Let me know if you are now able to submit 20MB files.

**Comment by ba66e77** [comment]
@vz3, no.  The memory limit is per-process so only impacts the submission of a single user.

**Comment by dangur** [comment]
Created https://www.drupal.org/project/drupal/issues/2939573
